  Header header
  uint32 message_id
  PulsesAtRpm motor_0
  PulsesAtRpm motor_1
  PulsesAtRpm motor_2
  PulsesAtRpm motor_3
  PulsesAtRpm motor_4
  PulsesAtRpm motor_5
  PulsesAtRpm motor_6
  PulsesAtRpm motor_7
  PulsesAtRpm motor_8
  PulsesAtRpm motor_9
